test/e2e/support/utils/get-url.js

```diff
@@ -6,6 +6,13 @@ import { URL } from 'url';
 
 import { WP_BASE_URL } from './config';
 
+/**
+ * Creates new URL by parsing base URL, WPPath and query string.
+ *
+ * @param {string} WPPath String to be serialized as pathname.
+ * @param {string} query String to be serialized as query portion of URL.
```

Comment:
`query` is optional so it should be `{?string}`.

Reply:
Adjusted, thx.